docs/activities.md

```diff
@@ -0,0 +1,96 @@
+# Sprint Activities List
+
+## Most popular
+
+### Wine tasting - 35€
+
+Dive deeper into the unique style of how wine is produced on Lanzarote.
+
+Choose four of your favourite styles from a portfolio of wines.
+
+A bite of artisanal cheese and charcuterie will accompany your personalized tasting. The experience takes about 60 minutes.
```

Comment:
Does this happen on-site, or a bus trip?

Reply:
bus trip to stravts